test_conformance/commonfns/test_mix.cpp

```diff
@@ -90,25 +109,48 @@ int verify_mix(const T *const inptrX, const T *const inptrY,
             int vi = i * veclen;
             for (int j = 0; j < veclen; ++j, ++vi)
             {
-                r = inptrX[vi] + ((inptrY[vi] - inptrX[vi]) * inptrA[i]);
-                delta = fabs(double(r - outptr[vi])) / r;
-                if (delta > MAX_ERR)
+                r = conv_to_dbl(inptrX[vi])
+                    + ((conv_to_dbl(inptrY[vi]) - conv_to_dbl(inptrX[vi]))
+                       * conv_to_dbl(inptrA[i]));
+                delta = fabs(double(r - conv_to_dbl(outptr[vi]))) / r;
+                if (!std::is_same<T, half>::value)
                 {
-                    log_error("{%d, element %d}) verification error: mix(%a, "
-                              "%a, %a) = *%a vs. %a\n",
-                              ii, j, inptrX[vi], inptrY[vi], inptrA[i], r,
-                              outptr[vi]);
-                    return -1;
+                    if (delta > MAX_ERR)
+                    {
+                        if (std::is_same<T, half>::value)
```

Comment:
Don't think we need this `if` condition any more given we check `if (!std::is_same<T, half>::value)` a few lines above

Reply:
of course, thanks!